### ISSUE TITLE
No Lang on <HTML>

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/css_layout_exercises_starter_files#44]
📛  **This is a serious violation of WCAG 2.0 Level A** 📛 

The `<HTML>` element should always have a lang attribute set. No exceptions. 

![](http://j4p.us/0X1t2x172Y2E/Screen%20Shot%202017-03-20%20at%2011.19.59%20AM.png)

If there are any questions or concerns resources can be provided but the TL;DR is that otherwise you wind up with [screen readers reading the page in English with a French accent](https://github.com/TryGhost/Casper/issues/286#issuecomment-281950302). 

## See Also
 - [<html> element must have a lang attribute](https://dequeuniversity.com/rules/axe/2.1/html-has-lang?application=axeChrome)
 - [It’s important to define the natural language of your document](https://medium.com/alistapart/writing-html-with-accessibility-in-mind-a62026493412#cd53)
